pom.xml

```
@@ -3,34 +3,45 @@
   <parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
-    <version>3.55</version>
+    <version>4.19</version>
+    <relativePath />
   </parent>
 
   <artifactId>implied-labels</artifactId>
-  <version>0.8-SNAPSHOT</version>
+  <version>${revision}${changelist}</version>
   <packaging>hpi</packaging>
   <name>Implied Labels Plugin</name>
   <description>Infer redundant labels automatically based on user declaration</description>
-  <url>http://wiki.jenkins-ci.org/display/JENKINS/Implied+Labels+Plugin</url>
+  <url>https://github.com/jenkinsci/implied-labels-plugin</url>
 
   <scm>
-    <connection>scm:git:git://github.com/jenkinsci/implied-labels-plugin.git</connection>
-    <developerConnection>scm:git:git@github.com:jenkinsci/implied-labels-plugin.git</developerConnection>
-    <url>https://github.com/jenkinsci/implied-labels-plugin</url>
-    <tag>HEAD</tag>
+    <connection>scm:git:git://github.com/${gitHubRepo}.git</connection>
+    <developerConnection>scm:git:git@github.com:${gitHubRepo}.git</developerConnection>
+    <url>https://github.com/${gitHubRepo}</url>
+    <tag>${scmTag}</tag>
   </scm>
 
+  <dependencyManagement>
+    <dependencies>
+      <dependency>
+        <groupId>io.jenkins.tools.bom</groupId>
+        <artifactId>bom-2.263.x</artifactId>
+        <version>887.vae9c8ac09ff7</version>
```

Use the newest Jenkins plugin BOM.
```suggestion
        <version>937.v51fde92016ed</version>
```